integration/docker/vsock_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package docker
+
+import (
+	"os"
+
+	"github.com/kata-containers/tests"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("vsock test", func() {
+	var (
+		args     []string
+		id       string = randomDockerName()
+		exitCode int
+		stdout   string
+	)
+
+	AfterEach(func() {
+		Expect(RemoveDockerContainer(id)).To(BeTrue())
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+	})
+
+	Context("when using vsock", func() {
+		It("should not create a kata-proxy process", func() {
+			val, ok := os.LookupEnv("USE_VSOCK")
```

Comment:
Read configuration file instead https://github.com/kata-containers/tests/blob/master/process.go#L67

btw you have to update https://github.com/kata-containers/tests/blob/master/config.go#L26 and add `use_vsock`